### ISSUE TITLE
[Regression] Rendering is broken on production

### ISSUE THREAD

**Issue by kptdobe** [adobe/project-helix.io#23]
After upgrade to helix-cli v0.9.1, the site renders partially:

* Nav (https://www.project-helix.io/SUMMARY.summary.html) renders a white page
* Index (https://www.project-helix.io/) renders only the title and commits history

This happens only on production, works fine with `hlx up`

**Comment by trieloff** [comment]
What do the logs say?

Index seems to have a strange include tag.

**Comment by tripodsan** [comment]
> This happens only on production, works fine with hlx up

not for me:

```
[hlx] error: Error while delivering resource /index.html - Error: Cannot find module 'request-promise'
```

**Comment by tripodsan** [comment]
I think it is cause by a discrepancy between the pipeline deploy in the docker and the ones in hlx.
another reason to use the default wsk contaiers.

**Comment by trieloff** [comment]
Yes, that might be the reason.

**Comment by kptdobe** [comment]
Fixed and deployed with helix-cli v0.10.15
